src/main/java/com/thealgorithms/searches/RecursiveBinarySearch.java

```diff
@@ -0,0 +1,52 @@
+// Code by Pronay Debnath
+// Created:- 1/10/2023
+// File Name should be RecursiveBinarySearch.java
+
+import java.util.*;
+public class RecursiveBinarySearch {
+
+    // Recursive binary search function
+    static int binsear(int a[], int left, int right, int target) {
```

Comment:
Can you make it generic? as all the other algos are.
https://github.com/TheAlgorithms/Java/blob/master/src/main/java/com/thealgorithms/searches/BinarySearch.java

Reply:
Also fine if you don't want to make it generic but it is better.